src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -439,6 +439,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 			NSAttributedString.Key.font: UIFont.preferredFont(forTextStyle: .largeTitle).scaledFont(size: 28, weight: .bold),
 			NSAttributedString.Key.foregroundColor: UIColor.enaColor(for: .textPrimary1)
 		]
+
+		appearance.shadowImage = UIImage()
```

Comment:
Do we want this? Does this change anything on iOS 123 and up?

Reply:
Also: maybe we can add some spacing above the segmented control, we have also done this for iOS 12 on other screens, see [here](https://github.com/corona-warn-app/cwa-app-ios/pull/1828)

Reply:
Spacing is already done - the missing shadow doesn't change anything on iOS 13

Reply:
> Do we want this? Does this change anything on iOS 13 and up?

iOS 12 has a drop shadow by default, iOS 13+ don't have one. This didn't changed anything in my tests